src/main/scala/nl/knaw/dans/easy/fedora2vault/DDM.scala

```diff
@@ -0,0 +1,238 @@
+import nl.knaw.dans.pf.language.emd.{ EasyMetadataImpl, EmdRights }
+import org.joda.time.DateTimeZone
+
+import scala.collection.JavaConverters._
+import scala.util.Try
+import scala.xml._
+
+object DDM extends DebugEnhancedLogging {
+  val schemaNameSpace: String = "http://easy.dans.knaw.nl/schemas/md/ddm/"
+  val schemaLocation: String = "https://easy.dans.knaw.nl/schemas/md/ddm/ddm.xsd"
+  val dansLicense = "http://dans.knaw.nl/en/about/organisation-and-policy/legal-information/DANSLicence.pdf"
+  val cc0 = "http://dans.knaw.nl/en/about/organisation-and-policy/legal-information/DANSLicence.pdf"
+
+  // for proper conversion of DateTime to date string by EMD's IsoDate/BasicDate
+  DateTimeZone.setDefault(DateTimeZone.forID("Europe/Amsterdam"))
+
+  def apply(emdNode: Node)(implicit fedoraProvider: FedoraProvider): Try[Elem] = {
+
+    for {
+      emd <- Try(new EmdUnmarshaller(classOf[EasyMetadataImpl]).unmarshal(emdNode.serialize))
+      fedoraIDs = emd.getEmdAudience.getDisciplines.asScala.map(_.getValue)
+      disciplines <- fedoraIDs.map(getAudience).collectResults
+    } yield {
+      //    println(new EmdMarshaller(emd).getXmlString)
+
+      val dateMap: Map[String, Iterable[Elem]] = getDateMap(emd)
+      val dateCreated = dateMap("created")
+      val dateAvailable = {
+        val elems = dateMap("available")
+        if (elems.isEmpty) dateCreated
+        else elems
+      }
+      <ddm:DDM
+        xmlns:dc="http://purl.org/dc/elements/1.1/"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:dct="http://purl.org/dc/terms/"
+        xmlns:dcx-dai="http://easy.dans.knaw.nl/schemas/dcx/dai/"
+        xmlns:dcx-gml="http://easy.dans.knaw.nl/schemas/dcx/gml/"
+        xmlns:gml="http://www.opengis.net/gml"
+        xmlns:abr="http://www.den.nl/standaard/166/Archeologisch-Basisregister/"
+        xmlns:ddm={schemaNameSpace}
+        xmlns:id-type="http://easy.dans.knaw.nl/schemas/vocab/identifier-type/"
+        xsi:schemaLocation={s"$schemaNameSpace $schemaLocation"}
+      >
+        <ddm:profile>
+          { emd.getEmdTitle.getDcTitle.asScala.map(bs => <dc:title xml:lang={ lang(bs) }>{ bs.getValue }</dc:title>) }
+          { emd.getEmdDescription.getDcDescription.asScala.map(bs => <dct:description xml:lang={ lang(bs) }>{ bs.getValue }</dct:description>) }
+          { emd.getEmdDescription.getTermsAbstract.asScala.map(bs => <dct:description xml:lang={ lang(bs) } descriptionType='Abstract'>{ bs.getValue }</dct:description>) }
+          { emd.getEmdDescription.getTermsTableOfContents.asScala.map(bs => <dct:description xml:lang={ lang(bs) } descriptionType='TableOfContent'>{ bs.getValue }</dct:description>) }
+          { /* instructions for reuse not specified as such in EMD */ }
+          { emd.getEmdCreator.getDcCreator.asScala.map(bs => <dc:creator eas:scheme={ bs.getScheme } eas:schemeId={ bs.getSchemeId } xml:lang={ lang(bs) }>{ bs.getValue }</dc:creator>) }
```

Comment:
i don't see what these scheme/schemeID do in a creator. Also, a language tag does not make sense to me in a name element.

```suggestion
          { emd.getEmdCreator.getDcCreator.asScala.map(bs => <dc:creator>{ bs.getValue }</dc:creator>) }
```